lib/report-portal-client.js

```diff
@@ -332,7 +333,8 @@ class RPClient {
         'filter.in.uuid': launchUUIds,
         'page.size': launchUUIds.length,
       });
-      const launchSearchUrl = `launch?${params.toString()}`;
+      const launchSearchUrl = mergeOptions.mode === 'DEBUG' || this.config.mode === 'DEBUG' ? 
+      `launch/mode?${params.toString()}` : `launch?${params.toString()}`;
```

Comment:
The `mode` provided via `mergeOptions` is intended to set the `mode` for the final merged launch.
So I assume it shouldn't affect the launches search `url` for already reported launches.

I think just checking `this.config.mode === 'DEBUG'` will suffice here.
In case another client instance is used to perform the launches merge operation, it will have its own `this.config.mode` to handle it correctly.

Reply:
Agreed